src/index.spec.ts

```diff
@@ -4,9 +4,195 @@ import {
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
 
-test('Testing time!', () => {
-  console.log('welcome');
+describe('isInteger tests', () => {
+  test('it should be valid with a positive number', () => {
```

Comment:
Inside a describe statement, it's more readable to use `it()` instead of `test()`, and starting with the string `should` in lowercase. For example:
```
it('should be valid with a positive number', () => {
```